gapis/service/path/path.proto

```diff
@@ -249,6 +249,6 @@ message Thumbnail {
     image.Format desired_format = 3;
 
     oneof object {
-        Command command = 4;
+        ResourceData resource = 4;
```

Comment:
Yes. It's even in its own commit. There is no atom that implements the Thumbnailer interface.